src/lib/y2storage/lvm_lv.rb

```diff
@@ -107,6 +122,23 @@ def stable_name?
     #   @return [Array<Disk>] all the logical volumes in the given devicegraph
     storage_class_forward :all, as: "LvmLv"
 
+    # Returns the thin pool holding a thin volume
+    #
+    # @return [LvmLv] the thin pool when dealing with a thin LV; nil otherwise
+    def thin_pool
+      lv_type.is?(:thin) ? storage_thin_pool : nil
```

Comment:
there is no `has_thin_pool` method in libstorage-ng?

Reply:
No. Or at least I didn't see it.